docker-compose.yml

```diff
@@ -3,6 +3,9 @@ version: '3'
 services:
   grader:
     image: apluslms/run-mooc-grader
+    entrypoint: >
+      bash -c "python3 manage.py pregenerate_exercises default;
+               python3 manage.py runserver 0.0.0.0:8080"
```

Comment:
Should I open a PR to https://github.com/A-plus-LMS/run-mooc-grader instead?

Reply:
Lets keep this as specific for the personalized template (reasoning in the large comment).